securedrop_client/app.py

```diff
@@ -150,7 +155,15 @@ def start_app(args, qt_args) -> None:
 
 
 def run() -> None:
+    config_file = "/usr/share/securedrop-client/client.ini"
     args, qt_args = arg_parser().parse_known_args()
+    if args.sdc_home == expand_to_absolute(DEFAULT_SDC_HOME) and \
+            os.path.exists(config_file):  # pragma: no cover
+        config = configparser.ConfigParser()
+        config.read(config_file)
+        args.sdc_home = config["client"]["homedir"]
```

Comment:
This is done to make sure that we use the whatever values are there in the config file. This part is to be used only after installing via debian package, and we can improve this in future.

Reply:
We have a hardcoded home directory in Qubes, so we're using that hard coded value to read a hardcoded config to then switch the home to something else hardcoded. This seems like a lot of indirection.